XenServerHealthCheck/ProjectInstaller.Designer.cs

```diff
@@ -33,9 +33,9 @@ private void InitializeComponent()
             // 
             // XenServerHealthCheckInstaller
             // 
-            this.XenServerHealthCheckInstaller.Description = Branding.COMPANY_NAME_SHORT + " " + Branding.PRODUCT_BRAND + " Health Check";
-            this.XenServerHealthCheckInstaller.DisplayName = Branding.COMPANY_NAME_SHORT + " " + Branding.PRODUCT_BRAND + " Health Check";
-            this.XenServerHealthCheckInstaller.ServiceName = Branding.PRODUCT_BRAND + "HealthCheck";
+            this.XenServerHealthCheckInstaller.Description = "[Citrix XenServer] Health Check";

```

Comment:
We should use the Branding variables here, because the `designer.cs` files are not getting rebranded, so [Citrix XenServer] will not be replaced when running the branding scripts

Reply:
I am going to introduce a "COMPANY_AND_PRODUCT_BRAND" variable in Branding.

For ServiceName, the base code doesn't have any space, but now with "Branding.PRODUCT_BRAND" we will introduce one. Any good idea?